src/kOS/Screen/TermWindow.cs

```diff
@@ -963,11 +961,20 @@ protected void CheckResizeDrag()
             }
         }
         
-        void ShowCharacterByAscii(char ch, int x, int y, bool reversingScreen, int charWidth, int charHeight, float brightness)
+        void DrawCursorAt(char ch, int x, int y, bool reversingScreen, int charWidth, int charHeight, float brightness)
```

Comment:
I didn't remove it because it sounded like the thing that could still be used for something important later.  It was complex enough that I don't want to forget how we did it.